src/components/base/JInput/JInput.js

```diff
@@ -140,7 +140,7 @@ class JInput extends PureComponent<Props> {
           <div className='lock'>
             <JIcon
               size='medium'
-              color='gray'
+              color={color}
```

Comment:
there is shouldn't be lock icon on blue screens

Reply:
In my opinion, all except this is optional